sig-list.md

```diff
@@ -49,6 +49,7 @@ When the need arises, a [new SIG can be created](sig-wg-lifecycle.md)
 |[Storage](sig-storage/README.md)|storage|* [Bradley Childs](https://github.com/childsb), Red Hat<br>* [Saad Ali](https://github.com/saad-ali), Google<br>|* [Slack](https://kubernetes.slack.com/messages/sig-storage)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-storage)|* Regular SIG Meeting: [Thursdays at 9:00 PT (Pacific Time) (biweekly)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit)<br>
 |[Testing](sig-testing/README.md)|testing|* [Erick Fejta](https://github.com/fejta), Google<br>* [Aaron Crickenberger](https://github.com/spiffxp), Google<br>* [Steve Kuznetsov](https://github.com/stevekuznetsov), Red Hat<br>* [Timothy St. Clair](https://github.com/timothysc), VMware<br>|* [Slack](https://kubernetes.slack.com/messages/sig-testing)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-testing)|* Regular SIG Meeting: [Tuesdays at 13:00 PT (Pacific Time) (weekly)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit)<br>* (kind) sigs.k8s.io/kind bi-weekly meeting: [Mondays at 11:00 PT (Pacific Time) (bi-weekly)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit)<br>* (testing-commons) Testing Commons: [Fridays at 07:30 PT (Pacific Time) (bi-weekly)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit)<br>
 |[UI](sig-ui/README.md)|ui|* [Dan Romlein](https://github.com/danielromlein), Google<br>* [Sebastian Florek](https://github.com/floreks), Loodse<br>* [Jeffrey Sica](https://github.com/jeefy), University of Michigan<br>* [Marcin Maciaszczyk](https://github.com/maciaszczykm), Loodse<br>|* [Slack](https://kubernetes.slack.com/messages/sig-ui)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-ui)|* Regular SIG Meeting: [Thursdays at 09:00 PT (Pacific Time) (bi-weekly)](https://groups.google.com/forum/#!forum/kubernetes-sig-ui)<br>
+|[Usability](sig-usability/README.md)|usability|* [Rajakavitha Kodhandapani](https://github.com/unknown), unknown<br>* [Vallery Lancey](https://github.com/unknown), unknown<br>* [Himanshu Pandey](https://github.com/unknown), unknown<br>|* [Slack](https://kubernetes.slack.com/messages/sig-usability)<br>* [Mailing List](TBD)|* Regular SIG Meeting: [TBDs at TBD PT (Pacific Time) (every third week)](tbd)<br>
```

Comment:
[Rajakavitha Kodhandapani](https://github.com/Rajakavitha1)

Reply:
If it's ok with you, I'd prefer to leave you as a lead. 

I have no particularly strong need to be chair, and I'd rather empower others in the community. I'll certainly help get this thing started as it is an area of interest for me.

Reply:
I'd be interested in being a lead.

Reply:
Also I think @gaby-moreno would be an excellent lead

Reply:
If folks want to re-arrange as we form up, I think that's great. SIG chair can be a bunch of work, so starting out with a larger number of chairs and then reducing down or rotating if there's not enough work seems like a good plan. 

@tashimi I added you.

@gaby-moreno if you want me to add you as a chair let me know.

Reply:
@brendanburns would love to, I also agree with your comment above.